spark/src/main/scala/org/apache/spark/sql/delta/DeltaErrors.scala

```diff
@@ -1599,6 +1599,12 @@ trait DeltaErrorsBase
       messageParameters = Array(DeltaSQLConf.DELTA_COMMIT_VALIDATION_ENABLED.key))
   }
 
+  def metadataAbsentForExistingCatalogTable(tableName: String, tablePath: String): Throwable = {
```

Comment:
Add a test in `DeltaErrorsSuite` for this error.

Reply:
Done